cinema/serializers.py

```diff
@@ -114,9 +114,9 @@ class TicketSerializer(serializers.ModelSerializer):
     def validate(self, attrs):
         data = super(TicketSerializer, self).validate(attrs=attrs)
         Ticket.validate_ticket(
-            attrs["row"], 
-            attrs["seat"], 
-            attrs["movie_session"].cinema_hall, 
+            attrs["row"],
+            attrs["seat"],
+            attrs["movie_session"].cinema_hall,
             ValidationError
         )
         return data
```

Comment:
In the `validate` method of `TicketSerializer`, ensure that `attrs` contains all the necessary keys (`row`, `seat`, `movie_session`) before accessing them. Otherwise, it might raise a `KeyError` if any of these keys are missing.